### ISSUE TITLE
wheel contains "tests" package?

### ISSUE THREAD

**Issue by jeroenp** [django-mptt/django-mptt#513]
Hi,

The django-mptt's wheel contains a "tests" package (just like FeinCMS, by the way) which gets installed in site-packages/. This can give issues with pip at some point when another package that also bundles a "tests" package tries to update/install: 

```
Operation not permitted: 'hopsa/lib/python2.7/site-packages/tests/settings.py'
```

I notice that you have an include in manifest.in and an exclude in find_packages. Yet somehow I end up with mptt's "tests" in my 0.8.6 install:  

```
$ mkvirtualenv hopsa
(hopsa) ~$ pip install django-mptt
(hopsa) ~$ cdsitepackages
(hopsa) site-packages$ pip freeze --all
django-mptt==0.8.6
pip==8.1.2
setuptools==26.0.0
wheel==0.29.0
(hopsa) site-packages$ ls
django_mptt-0.8.6.dist-info pip             setuptools-26.0.0.dist-info
easy_install.py         pip-8.1.2.dist-info     tests
easy_install.pyc        pkg_resources           wheel
mptt                setuptools          wheel-0.29.0.dist-info
```

(I also see __pycache__'s in the wheel, which you try to exclude in manifest.in)

**Comment by craigds** [comment]
Curious. previous: #453.

**Comment by rfleschenberg** [comment]
I can confirm the problem. It seems that the fix from #462 isn't working.

@craigds, since you asked about it in #453, here are two resources that I found very helpful so far.

* https://blog.ionelmc.ro/2014/05/25/python-packaging/
* https://github.com/ionelmc/cookiecutter-pylibrary/

**Comment by rfleschenberg** [comment]
It seems that the tar.gz on PyPI is ok, but the wheel is broken. It contains the `tests` package.

```
wget https://pypi.python.org/packages/a0/53/9753adb3ca98595e19c234905918b9167b08107fed5458802e68c77255d6/django_mptt-0.8.6-py2.py3-none-any.whl\#md5\=d5ee94ca58b04fb7ae56272bb7e773f2

unzip django_mptt-0.8.6-py2.py3-none-any.whl
```

It's just a guess, but maybe you had stale files lying around when building the wheel (see https://github.com/ionelmc/cookiecutter-pylibrary/#building-and-uploading). I suggest you clean up your build area and build the wheel again, that will probably fix it.
